sql/event_db_repository.cc

```diff
@@ -125,7 +125,8 @@ const TABLE_FIELD_TYPE event_table_fields[ET_FIELD_COUNT] =
     "'STRICT_ALL_TABLES','NO_ZERO_IN_DATE','NO_ZERO_DATE','INVALID_DATES',"
     "'ERROR_FOR_DIVISION_BY_ZERO','TRADITIONAL','NO_AUTO_CREATE_USER',"
     "'HIGH_NOT_PRECEDENCE','NO_ENGINE_SUBSTITUTION','PAD_CHAR_TO_FULL_LENGTH',"
-    "'EMPTY_STRING_IS_NULL','SIMULTANEOUS_ASSIGNMENT')") },
+    "'EMPTY_STRING_IS_NULL','SIMULTANEOUS_ASSIGNMENT',"
+    "'TIME_ROUND_FRACTIONAL','NO_NEW_TRANS_IN_TRANS')") },
```

Comment:
Discussed this with @PhysicsTing.

He is adding `TIME_ROUND_FRACTIONAL` here as well, because it was apparently omitted/forgotten in 4447a02cf13a49876001a40ca7db8fdedb731fd5.

Reply:
No, it wasn't, that would've broken too many tests. Here: https://github.com/MariaDB/server/commit/4447a02cf13a49876001a40ca7db8fdedb731fd5#diff-d4c0aa3fa92fea6f90d7f43915d55b441a5a741134c37e8b3f7909c7ebb9abdfR3356

Reply:
> No, it wasn't, that would've broken too many tests. Here: [4447a02#diff-d4c0aa3fa92fea6f90d7f43915d55b441a5a741134c37e8b3f7909c7ebb9abdfR3356](https://github.com/MariaDB/server/commit/4447a02cf13a49876001a40ca7db8fdedb731fd5#diff-d4c0aa3fa92fea6f90d7f43915d55b441a5a741134c37e8b3f7909c7ebb9abdfR3356)

@vuvova, you're showing that it was indeed added in `sys_vars.cc`. But it _wasn't_ added in _this_ file.

Reply:
Right, that's why I deleted my comment within minutes after posting it. Sorry for confusion. Indeed, `Table_check_intact::check` only requires that `set` type in `TABLE_FIELD_TYPE` is a **prefix** of what's actually in the table, so missing new value at the end of the `set` didn't trigger an error.